### PR TITLE
[WVD- MSIX App Attach] Created new API version for new type of MSIX Package 

### PR DESCRIPTION
**Pull request by shailahirji**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by shailahirji** [comment]
@zhenglaizhang @PhoenixHe-msft , tried to fix the checks for prettier and linter , but failed.
`[error] Cannot find module 'C:\azure-rest-api-specs\scripts\prettier-swagger-plugin' **Actually does exist**
[error] Require stack:
[error] - C:\Users\shhirji\AppData\Roaming\npm\node_modules\prettier\index.js
[error] - C:\Users\shhirji\AppData\Roaming\npm\node_modules\prettier\bin-prettier.js
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! azure-rest-api-specs-tests@0.1.0 prettier: `prettier "--write" "specification/desktopvirtualization/**/*.json"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the azure-rest-api-specs-tests@0.1.0 prettier script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:`


 Followed these instructions: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md#prettier-check.

File: specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2019-01-23-preview/examples/Application_Create.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "resourceGroupName": "resourceGroup1",
     "applicationGroupName": "applicationGroup1",
     "applicationName": "application1",
-    "api-version": "2019-01-23-preview",
+    "api-version": "2019-12-10-preview",
```
Comment (PhoenixHe-msft) [inline specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2019-01-23-preview/examples/Application_Create.json:7]: Typo?

Comment (shailahirji) [reply specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2019-01-23-preview/examples/Application_Create.json]: thanks for catching. fixed

**Review by PhoenixHe-NV — APPROVED** [review]

**Pull request merged**